src/css/application.css

```diff
@@ -329,14 +329,32 @@ body {
   flex: 1 1 auto;
   display: flex;
   flex-direction: row;
+  height: 100vh;
 }
 
 .environment-column {
   flex: 1 1 50%;
 }
 
+.container-label {
```

Comment:
@jwang1919 Popcode’s CSS uses a slightly modified version of the [Block Element Modifier convention](https://en.bem.info/methodology/quick-start/)—only difference is that it uses single dash `-` instead of double underscore `__` (honestly I should probably just refactor it to use BEM exactly, in order to be more approachable).

Anyway, under BEM, the existence of a `container-label` element implies the existence of a `container` top-level block component, which is not the case.

I think what probably makes the most sense is to define a top-level block component (`label` maybe? seems a little unspecific) that defines the non-positioning-related stuff here (background, color, padding, etc.) and then an `environment-paneLabel` that contains the positioning information (position, z-index, etc.). This follows the [“Mix” technique](https://en.bem.info/methodology/quick-start/#mix) of BEM. WDYT?